app/src/main/java/org/mozilla/fenix/home/recenttabs/RecentTabsListFeature.kt

```diff
@@ -54,13 +54,13 @@ sealed class RecentTab {
      * @param deviceDisplayName The device the tab was viewed on.
      * @param title The title of the tab.
      * @param url The url of the tab.
-     * @param previewImageUrl The url used to retrieve the preview image of the tab.
+     * @param iconUrl The url used to retrieve the icon of the tab.
```

Comment:
Just wanted to make sure that this change is intentional. `previewImageUrl` is a very distinct naming that follows the GV naming and is a preview (hero) image of the page and not an icon (hero image != icon). The preview (hero) image is parsed from the page's metadata (these are `<meta property="og:image">` tags).

Reply:
Thanks for double-checking.  think I was incorrect in the initial implementation by naming it `previewImageUrl`. After some testing, it was looking like it pointed at a favicon and not a hero image. The `iconUrl` naming convention matches the underlying data that is stored as a `TabEntry` in `RemoteTabsStorage` in A-C.

Reply:
👍